jupyterhub/app.py

```diff
@@ -2265,6 +2267,9 @@ def _log_cls(name, cls):
         self.init_tornado_settings()
         self.init_handlers()
         self.init_tornado_application()
+        HUB_STARTUP_DURATION_SECONDS.observe(
+            time.perf_counter() - hub_startup_start_time
+        )
```

Comment:
I believe it would be useful to have ``HUB_STARTUP_DURATION_SECONDS`` measure how much it takes to both *initialize* and *start* all the Hub components (including spawners).

Reply:
Done